tests/es6-timed-map.spec.ts

```diff
@@ -10,16 +10,29 @@ describe('Es6TimedMap', () => {
   test('can be created', () => expect(new Es6TimedMap()).toBeTruthy());
 
   // method/api tests for es6-map coverage
-  describe('delete', () => {
+  describe.only('delete', () => {
```

Comment:
```suggestion
  describe.only('delete', () => {
```

```suggestion
  describe('delete', () => {
```

Remove the `.only` so all tests are ran

Reply:
Updated, although I must have messed something up when rebasing on upstream because I'm seeing changes to package.json and package-lock.json that are already on master.